searchapp/urls.py

```diff
@@ -5,6 +5,6 @@
 
 urlpatterns = [
 
-    url(r'^(?P<city>[a-z]+)/(?P<category>[a-z]+)$', views.search, name='search'),
```

Comment:
@spyshiv sir, Also I will change my search form to post so the search query and token id and all will be not shown in the url..because all i need is local/delhi/bands.